jobs/scheduled/enqueue_reminders.rb

```diff
@@ -0,0 +1,38 @@
+module Jobs
+  class EnqueueReminders < Jobs::Scheduled
+    def execute(_args)
+      return if skip_enqueue?
+      user_ids.each { |id| Jobs.enqueue(:remind_user, user_id: id) }
+    end
+
+    private
+
+    def skip_enqueue?
+      SiteSetting.remind_assigns.nil? || SiteSetting.remind_assigns == 'never'
+    end
+
+    def user_ids
+      interval = reminder_interval_in_minutes(SiteSetting.remind_assigns)
+
+      TopicCustomField
+        .joins("LEFT JOIN users ON topic_custom_fields.value::INT = users.id")
+        .where("COALESCE(users.last_tasks_reminder, '2010-01-01') <= CURRENT_TIMESTAMP - ('1 MINUTE'::INTERVAL * ?)", interval)
```

Comment:
I kind of feel bummed to be reminding people of work that was just assigned to them today. 

I would add one more where clause here to ensure the thing was assigned prior to current_date - interval.